### ISSUE TITLE
PDF rendering error

### ISSUE THREAD

**Issue by jorgelondo** [bigbluebutton/bigbluebutton#21798]
I installed version 3 of bbb and when I did some tests I saw that it did not render a pdf well, this same pdf renders well in version 2.7
bbb 2.7.15
![BBB-2 7-Captura de pantalla 2024-11-28 122035](https://github.com/user-attachments/assets/aa452c77-4840-4507-8d80-818561a278d5)
bbb.3.0.0-beta.5 (1858)
![BBB-3Captura de pantalla 2024-11-28 122035](https://github.com/user-attachments/assets/2564fc96-d275-4bc7-abb2-a1d1f57bb9e1)

Thank you for your work

**Comment by jorgelondo** [comment]
Hi @antobinary, This is the slide 
[slide-error.pdf](https://github.com/user-attachments/files/17978600/slide-error.pdf)
Thanks

**Comment by antobinary** [comment]
tl;dr: So far it looks like the problem is in the imagemagick version we use on BigBlueButton 3.0

The slide has a high number of paths
`2024-12-04T01:31:44.848Z WARN  o.b.p.imp.SvgImageCreatorImp -  --analytics-- data={"numberOfImages":0,"filename":"slide-error.pdf","logCode":"potential_problem_with_svg","meetingId":"c2df639bc560a5818b61c6fc0d31584b99b85336-1733275884052","page":1,"fileExists":false,"message":"Potential problem with generated SVG","convertSuccess":true,"numberOfPaths":857,"presId":"7eb7f75428f4adf752221d64a8e76b0bbe7e67c5-1733275898016"}`
so we convert to PNG first and then to SVG.

The conversion PDF to PNG of that slide seems fine anywhere I tested.

`pdftocairo -r 300 -png -singlefile -scale-to-x 2048 -scale-to-y -1  -q -f 1 -l 1 /tmp/slide-error.pdf one`

one.png looks fine when I open it.

Then the step PNG to SVG is where things differ between BBB 2.7 and 3.0

`convert one.png -auto-orient two.svg`

two.svg when executed on BBB 3.0 looks like the black-white image you attached. Transparent background. I tried passing various flags to flatten, specify color, to no avail.

On BBB 2.7:
```
convert --version
Version: ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org
```

On BBB 3.0 by default we have `ImageMagick 6.9.11-60`
I tried installing the newest version but the results were the same. `ImageMagick 7.1.1-41 Q16-HDRI x86_64 d7d490223:20241115`

---

I tried bumping up the threshold

```
#----------------------------------------------------
# Maximum allowed number of place object tags in generated svg, if exceeded the conversion will fallback to full BMP (default 800)
placementsThreshold=800
```

I changed it to 900 in /etc/bigbluebutton/bbb-web.properties and **the slide displays fine now.**

I don't know why the PNG to SVG step behaves differently, remains to find out. I also am not spotting why we set the threshold to 800. But for time being, @jorgelondo please consider bumping it up from 800 to 900 and please let us know if that suffices.
